docs/policygen/README.md

```diff
@@ -31,7 +31,14 @@ are now generated by [tfengine](../tfengine/README.md) as Terraform configs. See
 ## Installation
 
 Download a pre-built
-[policygen binary](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/releases/)
+[policygen binary](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/releases/):
+
+```shell
+VERSION=v0.2.0
+wget https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/releases/download/${VERSION}/policygen_${VERSION}_linux-amd64 -O policygen
```

Comment:
I would recommend to put this in `/usr/local/bin/` by default.

Reply:
good point. done